Source: David Legrand <david.legrand@clever-cloud.com>
Upstream: No
Reason: Add `vet_test.v` as it leads to errors on ARMv8 platforms
---
 cmd/tools/vtest-self.v | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmd/tools/vtest-self.v b/cmd/tools/vtest-self.v
index e374b8c86..9de90f1e4 100644
--- a/cmd/tools/vtest-self.v
+++ b/cmd/tools/vtest-self.v
@@ -345,6 +345,7 @@ const skip_on_sandboxed_packaging = [
 	'vlib/v/gen/native/tests/native_test.v',
 	'vlib/v/compiler_errors_test.v',
 	'vlib/v/gen/c/coutput_test.v',
+	'cmd/tools/vvet/vet_test.v',
 ]
 
 // Note: musl misses openssl, thus the http tests can not be done there
-- 
2.39.3 (Apple Git-146)
